js/client/modules/@arangodb/testsuites/arangobench.js

```diff
@@ -56,119 +56,141 @@ const testPaths = {
 // //////////////////////////////////////////////////////////////////////////////
 
```

Comment:
are there any tests checking new functionality?

Reply:
it's because, at the moment, we don't have any infrastructure to test arangobench